### ISSUE TITLE
[BUG] Characters are getting dropped in chat messages on Android

### ISSUE THREAD

**Issue by ear-dev** [WideChat/Rocket.Chat#1365]
So far we have only seen this issue on Android mobile devices. It happens in both the mobile app and on the web app through a mobile browser.

After typing into the composer, if you send the message by hitting the little airplane icon, a bunch of the trailing characters in the message are dropped.  
- i.e. type "Hello" and only "H" shows up.  
- type "hello will this work" and something like "hello will this w" shows up.

If you send the message using the return button on the android keyboard we are not seeing this issue.  I think this points to some livechat code and compatibility with android perhaps.

**Comment by ear-dev** [comment]
- Possibly only chrome on Android. 
- State not getting updated properly.  
- Only happens when we use the send button.  
- If we type -> hide keyboard -> open keyboard -> push send, then it works. 

Note:  

- 2/28 new major version of myViasat deployed - web only.  This is when the issue was first reported. 
- 2/22 we upgraded to 5.x on Rocketchat.

**Comment by ear-dev** [comment]
@bhardwajaditya please prioritize this one.

**Comment by ear-dev** [comment]
@bhardwajaditya I have merged and will deploy this fix into prod tomorrow after it gets through our CI/CD.  I suspect however that the fix will not be applied to mobile app users unless they were to clear their app cache/data and re-login.  Likely, most users never do that.  Any thoughts about how to force our component/index.js to reload?  Or, since we are loading into a webview, it should refresh everytime?  I'm pretty sure I tested, and I had to clear all data/cache.
